cmd/templ/lspcmd/proxy/server.go

```diff
@@ -50,6 +53,8 @@ func NewServer(log *zap.Logger, target lsp.Server, cache *SourceMapCache, diagno
 	}
 }
 
+var preLoadURIs = []*lsp.DidOpenTextDocumentParams{}
```

Comment:
Could this be a field on the `Server` struct?

Reply:
Yeah. I can refactor that no problem.